tests/unit/models/summon.js

```diff
@@ -362,6 +362,78 @@ describe("Summon Model >", function() {
     });
   });
 
+  describe("summon model showJournalCoverImages method >", function() {
```

Comment:
Same for adding a couple tests for Summon's DOM elements behaving per the flag like I mentioned for the other two.